tests/data/validation/enum/enum.go

```diff
@@ -69,6 +69,16 @@ func (j *EnumMyBooleanTypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyBooleanTypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyBooleanTypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?

Reply:
interesting, that means that the compiled enum.go file wasn't ever validated before.

Reply:
correct. I started slowly adding that kind of coverage when I took ownership of maintaining this library, but it'll take some time.